src/data/transform.ts

```diff
@@ -641,5 +651,52 @@ export const convertToExcalidrawElements = (
       }
     }
   }
+
+  for (const [id, element] of elementsWithIds) {
```

Comment:
also let's add a comment here explaining we do this pass _after_ all other elements because we need the `frame.children` to be processed already